Cython/Compiler/Nodes.py

```diff
@@ -4888,6 +4889,19 @@ def analyse_declarations(self, env):
         self.class_result.analyse_annotations(cenv)
 
     def analyse_expressions(self, env):
+        if self.bases and not (self.bases.is_sequence_constructor and len(self.bases.args)==0):
+            from .ExprNodes import PythonCapiCallNode, CloneNode
+            # handle the Python 3.7 __mro_entries__ transformation
+            orig_bases = self.bases.analyse_expressions(env)
+            self.bases = PythonCapiCallNode(orig_bases.pos,
+                function_name = "__Pyx_PEP560update_bases",
+                func_type = PyrexTypes.CFuncType(
```

Comment:
The `CFuncType` looks constant. I'd keep it in the class rather than (re-)building it at need.